sympy/printing/latex.py

```diff
@@ -2199,7 +2199,11 @@ def translate(s):
         return s
 
 
-def latex(expr, **settings):
+def latex(expr, fold_frac_powers=False, fold_func_brackets=False,
+    fold_short_frac=None, inv_trig_style="abbreviated",
+    itex=False, ln_notation=False, long_frac_ratio=None,
+    mat_delim="[", mat_str=None, mode="plain", mul_symbol=None,
+    order=None, symbol_names={}):
```

Comment:
I think it would be better to have symbol_names default to `None`. Having mutables as defaults in the signature can be problematic.